### PR TITLE
Conversion to python3

### PR DESCRIPTION
**Pull request by adbar**
Hello,
I used your script for test purposes and converted it to python3, it is much easier to work with Unicode strings that way.
Best,
Adrien

### CODE REVIEW

**Comment by juditacs** [comment]
Thanks for the help.

Are you sure that `iter(dict.items())` is necessary instead of `dict.items()` ?

**Comment by adbar** [comment]
Yes, I think so, because in Python3 `dict.items()`is a view object, cf the doc:

> "Dictionary views can be iterated over to yield their respective data"
> https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects

**Comment by juditacs** [comment]
This still doesn't explain. You don't need to convert a view to an iterator explicitly, it is iterable already.

**Comment by adbar** [comment]
OK, I removed the iterators.

File: identify_lang.py [hunk 1]
```diff
@@ -14,13 +18,13 @@ def identify_input(models):
     if args.test_files:
         for fn in listdir(args.test_files):
             doc_path = path.join(args.test_files, fn)
-            with open(doc_path) as f:
-                clean = clean_text(f.read().decode('utf8', 'ignore'))[:args.test_cutoff]
+            with open(doc_path, 'r', encoding='utf-8') as f:
+                clean = clean_text(f.read())[:args.test_cutoff] # was f.read().decode('utf8', 'ignore')
```
Comment (juditacs) [inline identify_lang.py]: Can you please remove these comments?

**Comment by juditacs** [comment]
Thank you for your contribution.

**Pull request merged**